environments/echis/app-processes.yml

```diff
@@ -10,21 +10,21 @@ celery_processes:
     async_restore_queue:
   echis_server0:
     celery,export_download_queue,email_queue,case_import_queue:
-      concurrency: 1
+      concurrency: 8
       max_tasks_per_child: 5
     background_queue,case_rule_queue,analytics_queue:
-      concurrency: 1
-      max_tasks_per_child: 1
+      concurrency: 8
+      max_tasks_per_child: 5
     beat: {}
     celery_periodic:
-      concurrency: 1
+      concurrency: 4
     saved_exports_queue:
-      concurrency: 1
+      concurrency: 3
       max_tasks_per_child: 1
       optimize: True
     submission_reprocessing_queue:
       concurrency: 1
-    reminder_case_update_queue:
+    reminder_case_update_queue,ucr_indicator_queue,ucr_queue:
```

Comment:
It's better not to mix these queues, because these tasks require different pooling options. It's good to move it to any other queue that doesn't use `gevent` as pooling option